internal/database/mysql.go

```diff
@@ -120,59 +118,73 @@ func (db *MySQLDatabase) GetUser(id int) (user *internal.User) {
 	return
 }
 
-func (db *MySQLDatabase) getOpenID() (id int) {
-	err := db.underlyingDB.QueryRow("SELECT id FROM openid LIMIT 1").Scan(&id)
+func (db *MySQLDatabase) getOpenID() (id uint64, err error) {
+	err = db.underlyingDB.QueryRow("SELECT id FROM openid LIMIT 1").Scan(&id)
 	if err != nil && !errors.Is(err, sql.ErrNoRows) {
 		log.Println("Failed to query for an openid -", err)
+		return 0, errors.New("invalid openid")
 	}
-	return
+	return id, nil
```

Comment:
All functions should end in just "return" so that we know the happy case goes all the way to the end